src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,18 @@
+package calculator
+
+enum class Operator(val expression: String, val calculate: (Int, Int) -> Int) {
+    PLUS("+", { x, y -> x + y }),
+    MINUS("-", { x, y -> x - y }),
+    DIVIDE("/", { x, y -> x / y }),
+    MULTIPLY("*", { x, y -> x * y });
+
+    companion object {
+        fun isOperator(operator: String): Boolean {
+            return values().find { it.expression.equals(operator) } != null
```

Comment:
코틀린에서는 `==` (동등성) 연산이 내부적으로 `equals()` 메소드를 사용하고 있어서 아래 코드와 같이 간단하게 표현 가능한데요
```suggestion
            return values().find { it.expression == operator } != null
```

좀 더 간결해져서 가독성이 더 좋아지는 거 같다고 개인적으로 저는 느껴요~
참고차 리뷰 드립니다 🙇‍♂️